rpe/engines/python.py

```diff
@@ -83,6 +83,17 @@ def evaluate(self, resource):
 
         for policy_name, policy_cls in matched_policies.items():
             try:
+                if hasattr(policy_cls, "should_evaluate_policy"):
+                    should_evaluate_policy_res = policy_cls.should_evaluate_policy(
```

Comment:
can you move this and the below to a single line? don't think we need a variable for it. i.e if not policy_cls.should_evaluate_policy() : do something